recipes-kernel/linux/linux-imx-headers_6.6.bb

```diff
@@ -9,8 +9,8 @@ LIC_FILES_CHKSUM = "file://COPYING;md5=6bc538ed5bd9a7fc9398086aedcd7e46"
 
 SRC_URI = "git://github.com/nxp-imx/linux-imx.git;protocol=https;branch=${SRCBRANCH}"
 SRCBRANCH = "lf-6.6.y"
-LOCALVERSION = "-6.6.23-2.0.0"
-SRCREV = "b586a521770e508d1d440ccb085c7696b9d6d387"
+LOCALVERSION = "-6.6.36-2.1.0"
+SRCREV = "d23d64eea5111e1607efcce1d601834fceec92cb"
```

Comment:
Please split out the Linux update in a specific pull request.

Reply:
Done, https://github.com/Freescale/meta-freescale/pull/1991